skips.txt

```diff
@@ -28,80 +59,118 @@ array_api_tests/test_has_names.py::test_has_names[linalg-vecdot]
 array_api_tests/test_has_names.py::test_has_names[linalg-vector_norm]
 array_api_tests/test_has_names.py::test_has_names[linear_algebra-tensordot]
 array_api_tests/test_has_names.py::test_has_names[linear_algebra-vecdot]
-array_api_tests/test_has_names.py::test_has_names[linalg-cross]
-array_api_tests/test_has_names.py::test_has_names[linalg-cholesky]
-array_api_tests/test_linalg.py
+array_api_tests/test_has_names.py::test_has_names[manipulation-moveaxis]
+array_api_tests/test_has_names.py::test_has_names[manipulation-repeat]
+array_api_tests/test_has_names.py::test_has_names[manipulation-tile]
+array_api_tests/test_has_names.py::test_has_names[manipulation-unstack]
+array_api_tests/test_inspection_functions.py::test_array_namespace_info
+array_api_tests/test_inspection_functions.py::test_array_namespace_info_dtypes
+array_api_tests/test_linalg.py::test_tensordot
+array_api_tests/test_linalg.py::test_vecdot
+array_api_tests/test_manipulation_functions.py::test_concat
+array_api_tests/test_manipulation_functions.py::test_moveaxis
+array_api_tests/test_manipulation_functions.py::test_repeat
+array_api_tests/test_manipulation_functions.py::test_squeeze
+array_api_tests/test_manipulation_functions.py::test_tile
+array_api_tests/test_manipulation_functions.py::test_unstack
+array_api_tests/test_operators_and_elementwise_functions.py::test_acos
+array_api_tests/test_operators_and_elementwise_functions.py::test_acosh
+array_api_tests/test_operators_and_elementwise_functions.py::test_asin
+array_api_tests/test_operators_and_elementwise_functions.py::test_asinh
+array_api_tests/test_operators_and_elementwise_functions.py::test_atan
+array_api_tests/test_operators_and_elementwise_functions.py::test_atanh
+array_api_tests/test_operators_and_elementwise_functions.py::test_clip
+array_api_tests/test_operators_and_elementwise_functions.py::test_conj
+array_api_tests/test_operators_and_elementwise_functions.py::test_copysign
+array_api_tests/test_operators_and_elementwise_functions.py::test_cos
```

Comment:
These are quite a lot of additions. Is there something systematic that started failing?

Reply:
Yes, I also just noticed these. It is a hypothesis error. Possibly locally.

Reply:
I'm a little confused by what you meant with "possibly locally". Does this mean that some tests fail on your personal machine but not in CI?

Reply:
There are now fewer skips than there used to be when you first made the comment, but still too many. The test failures are due to divergences for extreme input values.